go/database/mpt/archive_trie_test.go

```diff
@@ -388,6 +389,11 @@ func TestArchiveTrie_CanProcessPrecomputedHashes(t *testing.T) {
 				t.Fatalf("failed to close resources: %v", err)
 			}
 
+			// create dummy codes.dat file
+			if err := writeCodes(nil, filepath.Join(archiveDir, "codes.dat")); err != nil {
```

Comment:
if it is expected the file must exists, it is perhaps a verification error if the file is not present.
Alternatively we have two options:
* create the file when the state is created and the file does not exist
* accept the missing file as an ok state, and read this as empty

Reply:
There is a [test](https://github.com/Fantom-foundation/Carmen/blob/67008e3801f9718da0284207e90d72b35ca294e5/go/database/mpt/verification_test.go#L70C6-L70C44) in the verification that explicitly asks for fail when the file does not exist. So creating the file is the logical option here.

Reply:
this is a bit different situation, the test you mention checks for forest files. Without these files, the forest cannot be loaded. 

But the MPT state without the codes file can be loaded. In this case an empty map is expected:
see https://github.com/Fantom-foundation/Carmen/blob/main/go/database/mpt/state.go#L472

for this reason I think verification should not fail on a missing file, and then you do not have to create the file here.

Reply:
Ic. I thought I should apply the logic from the test I mentioned to code file as well. Your comment make sense. Thanks!